### ISSUE TITLE
[CoE Starter Kit - QUESTION] Env Request | Create Approved Environment Flow

### ISSUE THREAD

**Issue by alzia-msft** [microsoft/coe-starter-kit#6043]
### Does this question already exist in our backlog?

- [X] I have checked and confirm this is a new question.

### What is your question?

Wondering if there is anyway I can create an environment using Power Platform Admin connector with D365 apps and set an environment URL? The Flow I Env Request | Create Approved Environment Flow has a connector **Create Database**, is there anyway we can somehow add D365 apps and possibly set an environment through the properties exposed by the connector? I have explored all Dataverse APIs and haven't see anyway to achieve it?

Thanks in advance.

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

3.34

### What app or flow are you having the issue with?

Env Request | Create Approved Environment Flow

### What method are you using to get inventory and telemetry?

Cloud flows

AB#652

**Comment by Jenefer-Monroe** [comment]
I dont believe there is a way today. Can add to the enhancement backlog for that feature area and they can drive with the connector owner.

**Comment by harini-2-y** [comment]
This issue is being closed as its scope is now included in a newly created consolidated enhancement request for centralized management of orphaned components (Apps, Flows, Connection References) in the CoE Starter Kit.

New Issue Reference: [[CoE Starter Kit - Feature]:https://github.com/https://github.com/microsoft/coe-starter-kit/issues/10333

Thank you for your contributions
